src/components/LandDetailPage/UnsetENSContentModal.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Button, Modal } from 'decentraland-ui'
+import { ENS } from 'modules/ens/types'
+import { Land } from 'modules/land/types'
+import { Link } from 'react-router-dom'
+import { locations } from 'routing/locations'
+
+type UnsetENSContentModalProps = {
+  land: Land
+  ens: ENS
+  open: boolean
+  onCancel: () => void
+  onConfirm: () => void
+}
```

Comment:
I think we can extract this to it's own [`.types.ts`](https://github.com/decentraland/standards/blob/master/standards/react-redux.md#4-component-types) file right?

Reply:
Done